week4/src/components/ImgCard.jsx

```diff
@@ -0,0 +1,97 @@
+import { WEATHER_TYPE } from "../constants/weather";
+import styled from "styled-components";
+import default_Img from "../assets/default_Img.gif";
+import Skeleton from "./Skeleton";
+
+const ImgCard = (props) => {
+  const { data, isLoading, isError } = props;
+
+  const temp = data.main && data.main.temp;
+  const feels_like = data.main && data.main.feels_like;
+  const maxTemp = data.main && data.main.temp_max;
+  const minTemp = data.main && data.main.temp_min;
+  const clouds = (data.clouds && data.clouds.all) * 100;
+
+  const weatherToday = JSON.stringify(
+    data.weather && data.weather[0].description
+  );
+  const filteredDay = WEATHER_TYPE.filter(
+    (it) => JSON.stringify(it.description) === weatherToday
+  );
+  const imgSrc = filteredDay && filteredDay[0] ? filteredDay[0].imgURL : "";
```

Comment:
여기두 요런식으로 옵셔널 체이닝을 쓰면 더 가독성 있을것같아!
```suggestion
  const imgSrc = filteredDay?.[0]?.imgURL || "";
```

Reply:
어 진짜 그렇게 쓸 생각은 못했었는데 옵셔널 체이닝 더 공부해봐야겠다..!! 참고할게잉!